Automattic/wp-calypso
# Add-on: Signup flow, implement step on Free sites to sell add-ons

#### Proposed Changes

* Adds the add-ons step to the signup flow if a Free plan is selected.
* Changes are hidden behind the `signup/add-ons` flag for all environments.
* Add-on cards are reused from #64289 

Addresses #64121 

<img width="480" alt="pricing - white version" src="https://user-images.githubusercontent.com/2749938/174993182-5cde6a81-b028-4256-880e-dfba1c3e2396.png">



#### Testing Instructions

* Go to `/start?flags=signup/add-ons`
* Select a Free plan
* You should see the Addons step
* Select a few add-ons and continue
* You should see the addons in your Shopping cart, on the Checkout page

* Selecting a Paid plan should not show you the Add-ons step

#### TODO
- [x] Replace subtitle copy placeholder
- [x] Show `Added to your plan` instead of `Included in your plan`
- [x] Show `Popular` label next to Premium themes
